core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -135,3 +124,24 @@
     width: 220px;
   }
 </style>
+
+<link rel="stylesheet" href="/third_party/static/perfectscrollbar-0.6.16/css/perfect-scrollbar.css" />
+<style>
+  .ps-container:hover > .ps-scrollbar-y-rail:hover > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+  .ps-container > .ps-scrollbar-y-rail > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+  .ps-container:hover > .ps-scrollbar-y-rail:hover {
+    background-color: #c5e1a5;
+  }
+  .ps-container.ps-in-scrolling.ps-y > .ps-scrollbar-y-rail > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+  .ps-container.ps-in-scrolling.ps-y:hover > .ps-scrollbar-y-rail > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+</style>
+
+<script src="/third_party/static/perfectscrollbar-0.6.16/js/perfect-scrollbar.js"></script>
```

Comment:
Looping @BenHenning in -- Ben, what are your thoughts on doing script/link imports in directives? They do reduce the page bloat, but I worry slightly that they might become hard to keep track of. More generally, where should we put third-party imports that only apply to specific pages?